custom/ewsghana/reports/specific_reports/dashboard_report.py

```diff
@@ -19,7 +20,8 @@ def report_config(self):
             domain=self.domain,
             startdate=self.datespan.startdate_utc,
             enddate=self.datespan.enddate_utc,
-            location_id=self.request.GET.get('location_id'),
+            location_id=self.request.GET.get('location_id') if self.request.GET.get('location_id')
+            else get_country_id(self.domain),
```

Comment:
it may be pep8 but i'd indent this more to differentiate it from the other args (or just allow a long line).

i think you could also use an `or` here to shorten the expression.